samples/sensor/tmp108/README.rst

```diff
@@ -0,0 +1,25 @@
+Title: Thermometer
```

Comment:
This page's content doesn't fit the usual template and doesn't build properly into HTML docs. Please revise.

You might look at samples/sensor/dht/README.rst for an example of how this is done elsewhere -- although it's up to @MaureenHelm as sensor maintainer to declare what the right template is for sensor sample documentation.

See https://docs.zephyrproject.org/latest/guides/docs/index.html#running-the-documentation-processors for instructions on building a docs change into HTML locally.